### PR TITLE
css files updated.

### PR DESCRIPTION
**Pull request by LaShundra**
The css files are updated.

### CODE REVIEW

**Review by LightningFrisk — APPROVED** [review]
tested and approved, no issues aside from the black dashboard text

File: src/main/resources/static/css/select.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+h1{
+font-weight: bold;
```
Comment (LightningFrisk) [inline src/main/resources/static/css/select.css:2]:
```suggestion
font-weight: bold;
color: white;
	font-size: 40px;
	text-shadow: -1px 1px 0 #000,
				  1px 1px 0 #000,
				 1px -1px 0 #000,
				-1px -1px 0 #000;
```

**Review by LightningFrisk — APPROVED** [review]

File: src/main/resources/static/css/select.css (hunk 1 above)
Comment (LaShundra) [reply src/main/resources/static/css/select.css:2]: Is this dealing with the menu at the top?

**Comment by LaShundra** [comment]
So, I am not able to approve my on request. 
Someone else needs to approve and merge

**Review by jghilliard — APPROVED** [review]

**Pull request merged**